### ISSUE TITLE
Generating a JSON plan with Client leads to NPE

### ISSUE THREAD

**Issue by twalthr** [stratosphere/stratosphere#878]
Calling `./bin/stratosphere info ... -p` with a new JAPI program leads to the following exception:

``` java
Exception in thread "main" java.lang.NullPointerException
    at eu.stratosphere.compiler.plandump.PlanJSONDumpGenerator.dumpOptimizerPlanAsJSON(PlanJSONDumpGenerator.java:99)
    at eu.stratosphere.compiler.plandump.PlanJSONDumpGenerator.getOptimizerPlanAsJSON(PlanJSONDumpGenerator.java:93)
    at eu.stratosphere.client.program.Client.getOptimizedPlanAsJson(Client.java:112)
    at eu.stratosphere.client.CliFrontend.info(CliFrontend.java:492)
    at eu.stratosphere.client.CliFrontend.parseParameters(CliFrontend.java:856)
    at eu.stratosphere.client.CliFrontend.main(CliFrontend.java:878)
```

**Comment by StephanEwen** [comment]
There is a fix already under test ;-)

**Comment by StephanEwen** [comment]
Fixed as part of abe88b7972c24b85338033042b07cba64ca91054
